lib/vector-core/src/tls/incoming.rs

```diff
@@ -446,6 +447,31 @@ impl From<X509> for CertificateMetadata {
     }
 }
 
+#[derive(Clone)]
```

Comment:
Did we remove this in a prior change? Why does enabling otel require us to do anything other then modify the Cargo.toml?

Reply:
Yes it was removed in some reason during updating to the Vector v0.33.0, i met the same error what was fixed here: https://github.com/ViaQ/vector/pull/198